src/idsk/components/subscription-form/subscription-form.js

```diff
@@ -0,0 +1,37 @@
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
+import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
+import { nodeListForEach } from '../../../../package/govuk/common'
```

Comment:
teraz som si vsimol, ze v projekte kombinujeme includovanie nodeListForEach z govuk a z idsk.. trebalo by to mozno refaktornut nech to bere vsade len z idsk konkretne nejak takto (import { nodeListForEach } from '../../common')